src/components/table/TableBasic.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <el-table
+    :data="tableData"
+    style="width: 100%"
+  >
+    <el-table-column
```

Comment:
is it possible to expose the table columns? Since we don't know what they will be and will be dependent on the data added to the data table